src/network/networkd-link.c

```diff
@@ -2378,9 +2378,26 @@ static int link_set_ipv6_mtu(Link *link) {
         if (link->network->ipv6_mtu == 0)
                 return 0;
 
+        /* IPv6 protocol requires a minimum MTU of IPV6_MTU_MIN(1280) bytes
+         * on the interface. Bump up MTU bytes to IPV6_MTU_MIN. */
+        if (link->network->ipv6_mtu < IPV6_MIN_MTU) {
+
+                log_link_warning(link, "Bumping IPv6 MTU to " STRINGIFY(IPV6_MIN_MTU) ", as "
+                                 "IPv6 requires a minimum MTU of " STRINGIFY(IPV6_MIN_MTU) " bytes: %m");
```

Comment:
Maybe:
```C
log_link_notice("Bumping IPv6 MTU to "STRINGIFY(IPV6_MIN_MTU)"  byte minimum required").
```
%m doesn't make sense without an error number.

Reply:
oops!  my cut-and-paste error, sorry.  will update that.